### DISCUSSION TITLE
Error when installing the module inside of a brand new Expo managed project

### DISCUSSION THREAD

**Discussion by peterpelle** [invertase/react-native-firebase#6571] [Q&A]
I followed https://rnfirebase.io/#managed-workflow guide on how to install Firebase. I created a new Expo app with:

npx create-react-native-app -t with-dev-client  

As written in the guide. I then did

npx expo install @react-native-firebase/app 
npx expo install @react-native-firebase/auth 

I followed the guide, added the GoogleServices to the root of the directory and copied the app.json edits into the correct position.

I then installed @react-native-firebase/perf, /crashlytics and expo-build-properties and added the use_frameworks.

Next I finished the guide with npx expo prebuild --clean.

Which gave this warning:

    (node:28580) [DEP0148] DeprecationWarning: Use of deprecated folder mapping

    "./" in the "exports" field module resolution of the package at

    C:\Users\username\expoproject\node_modules\@expo\devcert\node_modules\tslib\package.json.

    Update this package.json to use a subpath pattern like "./*".

    (Use `node --trace-deprecation ...` to show where the warning was created)

Having nothing in the project except 2 View components and a Button that creates an account with createUserWithEmailAndPassword.

This is when I am presented with the error:

    Error: You attempted to use a firebase module that's not installed on your Android project by calling firebase.app(). 
Ensure you have:imported the 'io.invertase.firebase.app.ReactNativeFirebaseAppPackage' module in your 'MainApplication.java' file.Added the 'new ReactNativeFirebaseAppPackage()' line inside of the RN 'getPackages()' method list. See http://invertase.link/android for full setup instructions.

My App.js:

```
import auth from '@react-native-firebase/auth';  

export default function App() {     
  function handleSignUp() {         
    auth()             
      .createUserWithEmailAndPassword(                 
        'ExampleEmail@gmail.com',                 
        'SuperSecretPassword!'
      )            
      .then(() => {                 
        console.log('User account created & signed in!');             
      }); 
...      
<Button title='Open Dev Menu' onPress={handleSignUp} />

```
Did I mess up on the way, or is there more steps than the guide suggests, and what can I do to fix it?

This was tested on an Android Emulator, launched by pressing 'a' in the terminal.

I appreciate any help!

Edit: Clarified a few things and provided more information.

app.json:

```
{
    "expo": {
        "name": "ExpoProject",
        "slug": "expoproject",
        "scheme": "expoproject",
        "version": "1.0.0",
        "orientation": "portrait",
        "icon": "./assets/icon.png",
        "splash": {
            "image": "./assets/splash.png",
            "resizeMode": "contain",
            "backgroundColor": "#ffffff"
        },
        "updates": {
        "fallbackToCacheTimeout": 0
        },
        "assetBundlePatterns": ["**/*"],
        "ios": {
            "googleServicesFile": "./GoogleService-Info.plist",
            "supportsTablet": true,
            "bundleIdentifier": "com.expoproject.ExpoProject"
        },
        "android": {
            "googleServicesFile": "./google-services.json",
            "adaptiveIcon": {
                "foregroundImage": "./assets/adaptive-icon.png",
                "backgroundColor": "#FFFFFF"
            },
            "package": "com.expoproject"
        },
        "web": {
            "favicon": "./assets/favicon.png"
        },
        "plugins": [
            "@react-native-firebase/app",
            [
                "expo-build-properties",
                {
                    "ios": {
                        "useFrameworks": "static"
                    }
                }
            ]
        ]
    }
}
```

EDIT: 
Problem was Expo Go. Switching to a custom build solved it.

**Comment by coderpr0grammer** [comment]
I'm having the same issue. Maybe try adding "@react-native-firebase/auth" into your plugins?
